src/main/java/StringCalculator/StringSum.java

```diff
@@ -0,0 +1,65 @@
+package StringCalculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringSum {
+
+    private String formula;
+    private String delimiter;
+
+    public StringSum(String formula) {
+        this.formula = formula;
+        delimiter = ",|:";
+    }
+
+    public String customDelimiter() {
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(formula);
+        String customDelimiter = null;
+        if (m.find()) {
+            customDelimiter = m.group(1);
+            formula =  m.group(2);
+            delimiter += "|" + customDelimiter;
+        }
+        return customDelimiter;
+    }
+
+    public String[] splitByDelimiter() {
+        String[] tokens = formula.split(delimiter);
+        return tokens;
+    }
```

Comment:
inline 처리해주어도 좋을 것 같아요~

```suggestion
    public String[] splitByDelimiter() {
        return formula.split(delimiter);
    }
```